protocol/avdl/config.avdl

```diff
@@ -13,6 +13,21 @@ protocol config {
 
   GetCurrentStatusRes getCurrentStatus(int sessionID);
 
+  record ExtendedStatus {
+    boolean standalone;
+    boolean passphraseStreamCached;
+    DeviceID deviceID;
+    string deviceName;
+    string deviceStatus;
+    string logDir;
+    boolean desktopUIConnected;
```

Comment:
There are a lot of callers of GetCurrentStatus now...I didn't want to mess with the protocol.